docs/src/pages/demos/tables/EnhancedTable.js

```diff
@@ -148,6 +154,118 @@ EnhancedTableToolbar.propTypes = {
 
 EnhancedTableToolbar = withStyles(toolbarStyles)(EnhancedTableToolbar);
 
+const footerStyles = theme => ({
+  cell: {
+    padding: '0 !important',
```

Comment:
What's the important for? It's the first introduction of such pattern. It shouldn't be needed. And if we do, something is wrong.

Reply:
Maybe something is wrong, because something forced the cell to keep having padding even if I set padding to zero here. `!important` fixed this. I'll look into that later.

Reply:
I can have a look at this.

Reply:
I'd really appreciate your help on this, I'll focus on the other things in the meantime. :+1: